docs/rst/manual/reference/workflows/internal.rst

```diff
@@ -0,0 +1,83 @@
+Internal Workflows
+==================
+
+These jobs invoke a function in the address space of the ERT program
+itself; i.e. they are run as part of the running ERT process - and can
+in principle do anything that ERT can do itself. There are two two
+varieties of the internal workflow jobs:
+
+
+
+Invoke a pre exported function
+------------------------------
+
+This is the simplest, where you can invoke a a predefined ERT
+function. The function must already have been marked as *exported* in
+the ert code base. The list of predefined workflow jobs based on this
+method can be found here: :ref:`built_in_workflow_jobs`. Marking a new
+function as exported is quite simple, but it requires changes to the
+core code and a new version must be installed.
+
+.. _ert_script:
+
+Run a Python Script
+-------------------
+
+If you are using one of the Python based frontends, *gert* or
+*erthsell*, you can write your own Python script which is run as part
+of the existing process. By using the full ert Python api you get
+access to powerful customization/automization features. Below is an
```

Comment:
This sentence might be an understatement ;) Also, `automization` is misspelled, should be `automation`